workflows/cloudify_system_workflows/plugins.py

```diff
@@ -21,7 +21,15 @@
 
 @workflow(system_wide=True)
 def install(ctx, plugin, **_):
-    return _operate_on_plugin(ctx, plugin, 'install')
+    try:
+        execution_result = _operate_on_plugin(ctx, plugin, 'install')
+    except Exception as e:
+        client = get_rest_client()
+        plugins = client.plugins.list(id=plugin['id'])
```

Comment:
why not plugin.get?

Reply:
Just to make sure that this doesn't fail on plugin.get if the plugin entry doesn't exist. This way the correct error will be raised